usage.txt

```diff
@@ -8,3 +8,4 @@ Usage
 
   Optional options:
     out:               path to output the installer (path must exist)
+    nsiTemplate:       path to override the installer.nsi template for win platform (path must exist)
```

Comment:
yeah, makes sense. the path would be relative to the place you run the command / script from then, like with other asset paths, and not to the config (which may be a bit confusing for the user, but it's the way it is and still is fairly simple to understand)